src/app/work-item/work-item-list/work-item-list.module.ts

```diff
@@ -12,6 +13,7 @@ import { WorkItemListEntryComponent } from './work-item-list-entry/work-item-lis
   imports:      [
     CommonModule, 
     WorkItemQuickAddModule,
+    Ng2Dropdown,
     DropdownModule,
```

Comment:
Do you still need the DropdownModule if you replaced it with a new name?

If you need the old name then don't make a new one until you can change both.